blueprints/ember-flexure/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
By convention, addons should have a default blueprint with the same name as the addon. It's also useful for installing the addon when developing locally via `npm link`